sklearn/datasets/_samples_generator.py

```diff
@@ -839,7 +839,7 @@ def make_blobs(
         See :term:`Glossary <random_state>`.
 
     return_centers : bool, default=False
-        If True, then return the centers of each cluster
+        If True, then return the centers of each cluster.
```

Comment:
These changes to `make_blobs` are not needed are they were merged in https://github.com/scikit-learn/scikit-learn/pull/22342.